### PR TITLE
Feature/connect tool endpoints

### PR DESCRIPTION
**Pull request by yashgajaria**
## Notion ticket link

[Connect 'tool' GET route to dashboard](https://www.notion.so/uwblueprintexecs/Connect-tool-GET-route-to-dashboard-dccfbe61584042a090ca927b3165d6d2)

## Implementation description
* Used axios to call the api endpoint for getting all the tools
* Needed to add `createdBy` key to the database model/schema and seeding script for tools because it wasn't initially there

## Steps to test
1. Re-seed your database with `yarn seed` to get the createdBy field for tools
2. Check `admin/dashboard/tools` to see the tools loaded from the database 

## What should reviewers focus on?
* Seeing the tools on `admin/dashboard/tools` 
* The reseeding/editing of the tools schema for the `createdBy` string of arrays that was added

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: pages/admin/dashboard/tools.tsx
Comment (mkanoria) [inline pages/admin/dashboard/tools.tsx:43]: Not sure if logging to console is the best idea but if we do we should also log the actual error to the console

**Review by mkanoria — APPROVED** [review]
Small comment, but looks good over all! Approved so that you can merge once finalized

File: pages/admin/dashboard/tools.tsx
Comment (yashgajaria) [reply pages/admin/dashboard/tools.tsx:43]: I think we don't have any error handling in place yet, will log the actual error to the console and keep a todo comment for handling the error

**Comment by mkanoria** [comment]
Looks good to me! Would get another approval during the work session and then merge this in!

File: database/models/tool.ts
Comment (daniel-sm-yu) [inline database/models/tool.ts:19]: when are there multiple "creators" of a tool?

File: pages/admin/dashboard/tools.tsx
Comment (daniel-sm-yu) [inline pages/admin/dashboard/tools.tsx:11]: is the comment necessary?

File: pages/admin/dashboard/tools.tsx
Comment (daniel-sm-yu) [inline pages/admin/dashboard/tools.tsx:34]: `/api/tool` calls `getAll`, so `url: "/api/tool"` would work

File: database/models/tool.ts
Comment (yashgajaria) [reply database/models/tool.ts:19]: not totally sure if we have the use case of multiple creators of a tool- just followed the schema plan from the[ database schema on Notion](https://www.notion.so/uwblueprintexecs/Database-Schema-a256105a04c946f7a69d26613b2388bd)

Comment (daniel-sm-yu) [reply database/models/tool.ts:19]: sounds good!

File: pages/admin/dashboard/tools.tsx
Comment (yashgajaria) [reply pages/admin/dashboard/tools.tsx]: Updated url to `/api/tool`

**Pull request merged**
